src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockWithEmoji.java

```diff
@@ -0,0 +1,52 @@
+/*
+EmptyBlock
+option = TEXT
+tokens = LITERAL_WHILE, LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, LITERAL_DO, LITERAL_IF, \
+         LITERAL_ELSE, LITERAL_FOR, INSTANCE_INIT, STATIC_INIT, LITERAL_SWITCH, LITERAL_DEFAULT, \
+         LITERAL_CASE
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.emptyblock;
+
+public class InputEmptyBlockWithEmoji {
+    // violation below 'Empty STATIC_INIT block'
+    static {
+
+    }
+    static {
+        String c = "🎄";
+    }
+    public void fooMethod()
+    {
+        String a = "12🤩";
+        String b = "";
+        if (a == "🤩12🧐🧐") { } // violation 'Empty if block'
```

Comment:
weird side effect, symbols are not even inside next to curly braces, but close enough to confuse Check